Firmware/ultralcd.cpp

```diff
@@ -2992,23 +2992,15 @@ static const char* lcd_display_message_fullscreen_nonBlocking_P(const char *msg)
 {
     lcd_set_cursor(0, 0);
     const char *msgend = msg;
-    uint8_t row = 0;
     bool multi_screen = false;
-    for (; row < 4; ++ row) {
-        while (pgm_is_whitespace(msg))
```

Comment:
I'm not sure it's ok to remove this. What if the word on the previous line finishes with the last character on the last column of the screen? Wouldn't that cause the next line to have a useless space at the beginning?

Reply:
You're correct. See images below:

`lcd_display_message_fullscreen_P(_i("Averyveryverlongword Press the knob to pr eheat nozzle and continue."));`

MK3 branch test:
![IMG_2731](https://user-images.githubusercontent.com/8218499/206897317-bee493f9-2065-4573-96ab-df1ef0f6f768.jpg)


PR branch test:
![IMG_2729](https://user-images.githubusercontent.com/8218499/206897168-ccb6ecfc-96a2-4708-8986-c49b5ea01316.jpg)

Reply:
Fixed in aca9e79